docs/testing/playwright/02-e2e-testing.md

```diff
@@ -0,0 +1,133 @@
+---
+title: End-to-End Testing
+sidebar_label: E2E Testing
+---
+
+When it comes to writing end-to-end tests using the Stencil Playwright adapter, the best advice we can give is to leverage
+[Playwright documentation](https://playwright.dev/docs/writing-tests). The adapter is set up in a way so that developers will use the
+same public APIs with as little Stencil nuances as possible.
+
+## Writing Tests
+
+As far as writing the actual tests goes, the most important thing to be aware of for tests to run correctly is to import the `test` function from
+`@stencil/playwright`, **not** directly from `@playwright/test`:
+
+```ts
+// THIS IS CORRECT
+import { test } from '@stencil/playwright';
+
+// THIS IS NOT!!
+// import { test } from '@playwright/test';
+```
```

Comment:
I know it might be more work, but is there any way we could have a minimal, smallest-possible 'hello world' type test here? I think with any tutorial like this one of the most important things is getting the reader to a state where they have _something_ running as quickly as possible.

But when I'm reading through it's not immediately apparent what I want to do here to go forward. If I go check the ['Writing Tests'](https://playwright.dev/docs/writing-tests) page on the Playwright docs I see this:

```ts
import { test, expect } from '@playwright/test';

test('has title', async ({ page }) => {
  await page.goto('https://playwright.dev/');

  // Expect a title "to contain" a substring.
  await expect(page).toHaveTitle(/Playwright/);
});
```

but I'm not sure how exactly I'd want to adapt that to make it work with our setup, beyond that it looks like I need to `import { test } from '@stencil/playwright'`. But what do I want to pass to `page.goto`?

Reply:
Actually thinking about it more, I think it might be good to have a minimal test on the first page ('Overview') as the last step, what do you think?

Reply:
I want to create a live code example on StackBlitz we can link out to for a working example

Reply:
makes sense!